backend/layers/processing/process_download.py

```diff
@@ -23,14 +23,14 @@
 from backend.layers.thirdparty.step_function_provider import StepFunctionProvider
 from backend.layers.thirdparty.uri_provider import UriProviderInterface
 
-MEMORY_MODIFIER = 1.1  # add 10% overhead
-MEMORY_PER_VCPU = 4000
-MIN_MEMORY_MB = 4000
-MIN_VCPU = 1
-# The largest machine we are allocating is r5ad.2xlarge. This machine has 64GB of memory and 16 vCPUs.
-MAX_MEMORY_MB = 64000
-MAX_VCPU = 16
-SWAP_MEMORY_MB = 300000
+MEMORY_MODIFIER = 2  # add % overhead
```

Comment:
nit: what % overhead? 100?

Reply:
in this case it would be 200%. Would "add x% additional memory"

Reply:
got it. Maybe good to just be explicit with 'adds (x*100)% memory overhead'

Reply:
```suggestion
MEMORY_MODIFIER = 2  # adds (x*100)% memory overhead
```